src/Microsoft.AspNetCore.Blazor.Server/Builder/BlazorCircuitsApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Blazor.Browser.Rendering;
+using Microsoft.AspNetCore.Blazor.Server.Circuits;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// Extension methods to configure an <see cref="IApplicationBuilder"/> for Blazor circuits.
+    /// </summary>
+    public static class BlazorCircuitsApplicationBuilderExtensions
```

Comment:
Did we get to a conclusion about branding for this stuff?

Reply:
I like the term "circuit" to represent the stateful aspect of a live-updating UI connection and would be totally happy to keep it.

For the services that people are configuring though, terms like `UseServerSideBlazor` and `AddServerSideBlazor` feel more natural to me, in the sense that it's what we keep saying by default in our discussions and pretty unambiguously reflect the type of UI mechanism we're offering.

Reply:
Fixed